src/BaGet.Protocol/Models/PackageMetadata.cs

```diff
@@ -114,6 +114,12 @@ public class PackageMetadata
         [JsonProperty("title")]
         public string Title { get; set; }
 
+        /// <summary>
+        /// The package's release notes.
+        /// </summary>
+        [JsonProperty("releaseNotes")]
+        public string ReleaseNotes { get; set; }
```

Comment:
All the properties on `PackageMetadata` are part of the official NuGet protocol (see [this](https://docs.microsoft.com/en-us/nuget/api/registration-base-url-resource#registration-page-object)).

Please move this to `BaGetPackageMetadata`, as that is where all of BaGet's custom additions to the protocol are placed. See: https://github.com/loic-sharma/BaGet/blob/46acdaae2b78d820e30ac9a84ca5e23700444bf9/src/BaGet.Core/Metadata/BaGetPackageMetadata.cs#L8